### DISCUSSION TITLE
New docs are displayed in previous versions of the site.

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#5657] [Q&A]
I have enabled versioning on portal, so i have the same documentation available in two versions, one is stable and versioned, the other in development status. After creating a document in the development version, i have a linkl to access it in the versioned version, even if no document exists there. Even worse, if i click on it i see the document which is in development as it is existing in the actual versioned documentation site.

Is this something that needs to be configured anywere? Like a page where users are redirected if a document does not exists?

**Comment by slorber** [comment]
It's complicated to help you without current config, doc sources (at least md used for linking), live examples that we can look at...

We recommend using relative md file paths for docs file linking so that one doc link to the same doc of the same version. Double-check that first

https://docusaurus.io/docs/next/docs-markdown-features#referencing-other-documents


Keep in mind absolute links are kept untouched by Docusaurus, if you link to `/docs/myDoc`, it won't add a version magically for you (ie it won't link to `/docs/<version>/myDoc`)
